doc/source/examples/incompressible-flow/incompressible-flow.rst

```diff
@@ -63,6 +64,8 @@ Incompressible Flow
       incompressible_1_8 [label="Taylor-Couette Flow Using \nNitsche Immersed Boundary", href="https://chaos-polymtl.github.io/lethe/documentation/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.html", tooltip="Taylor-Couette flow using Nitsche immersed boundary"];
 
       incompressible_1_9 [label="Sudden-Expansion Flow", href="https://chaos-polymtl.github.io/lethe/documentation/examples/incompressible-flow/2d-sudden-expansion-flow/2d-sudden-expansion-flow.html"];
+      
+      incompressible_1_10 [label="Method of Manufactured Solutions", href="https://chaos-polymtl.github.io/lethe/documentation/examples/incompressible-flow/2d-mms/2d-mms.html", tooltip="Method of Manufactured Solutions"];
```

Comment:
I would move this example above. Maybe after the Taylor-Couette one for example. it is quite a basic example so it could be earlier in the list.